bridge/src/config.rs

```diff
@@ -1,20 +1,28 @@
-use std::path::{Path, PathBuf};
-use std::fs;
+use std::fs::File;
 use std::io::Read;
+use std::path::{Path, PathBuf};
+use std::str::FromStr;
+use std::sync::{Arc, RwLock};
 use std::time::Duration;
+
 #[cfg(feature = "deploy")]
 use rustc_hex::FromHex;
-use web3::types::Address;
+use toml;
+use web3::types::{Address, U256};
 #[cfg(feature = "deploy")]
 use web3::types::Bytes;
-use error::{ResultExt, Error};
-use {toml};
+
+use error::{Error, ResultExt};
 
 const DEFAULT_POLL_INTERVAL: u64 = 1;
 const DEFAULT_CONFIRMATIONS: usize = 12;
 const DEFAULT_TIMEOUT: u64 = 3600;
 const DEFAULT_RPC_PORT: u16 = 8545;
 const DEFAULT_CONCURRENCY: usize = 100;
+const DEFAULT_GAS_PRICE_ORACLE_URL: &str = "https://gasprice.poa.network";
```

Comment:
Instead of falling back to a default Oracle, I'd suggest to make oracles optional (if not specified, falls back to hard-coded value), and couple this with the suggestion of `jmespath` expression -- this way the bridge is flexible enough to accommodate a range of changes in the future.

Reply:
Ok, so there would be options in the config .toml file for:
```
home/foreign.gas_price_oracle_url
home/foreign.gas_price_oracle_expr
home/foreign.default_gas_price
```
In the situation where a user does not provide a value for `gas_price_oracle_url` and `gas_price_oracle_expr`, we default price to `default_gas_price` if it is provided in the toml file, or else default to a hardcoded const in config.rs?